core/server/worker/src/main/java/alluxio/worker/dora/PagedDoraWorker.java

```diff
@@ -274,14 +282,17 @@ public FileInfo getFileInfo(String ufsFullPath, GetStatusPOptions options) throw
     if (status == null) {
       // The requested FileStatus is not present in memory cache.
       // Let's try to query local persistent DoraMetaStore.
-      Optional<DoraMeta.FileStatus> fs = mMetaStore.getDoraMeta(ufsFullPath);
+      Optional<DoraMeta.FileStatus> fs = mMetaStore != null ? mMetaStore.getDoraMeta(ufsFullPath) :
```

Comment:
any concurrency issue?

Reply:
I initiated a discussion in Dora group on this issue.

This function may be called concurrently.
But this should not introduce severe problems, excepting probably calling getStatus() multiple times.
The same Metadata will be overwritten but no harm.